### PR TITLE
Add documentation to Time::Span and Time::Kind

### PR DESCRIPTION
**Pull request by adlerhsieh**

### CODE REVIEW

File: src/time/span.cr [hunk 1]
```diff
@@ -1,3 +1,39 @@
+# `Span` represents one period of time. 
+#
+# A `Span` initializes with a specified period. Different numbers of argumnets generates a `Span` in different length. Check all `#new` methods for details.
```
Comment (ysbaddaden) [inline src/time/span.cr]: Typo: arguments.

Also maybe refer to Time::Span instead of Span?

File: src/time/time.cr [hunk 2]
```diff
@@ -83,6 +83,27 @@ struct Time
   DP100 = 36524
   DP4   = 1461
 
+  # `Kind` represents a specified time zone. It can either be *Local*, *Utc*, or *Unspecified*.
```
Comment (ysbaddaden) [inline src/time/time.cr]: No need to list the enum values, that's docs' job.

File: src/time/time.cr [hunk 3]
```diff
@@ -253,14 +274,17 @@ struct Time
     year_month_day_day_year[3]
   end
 
+  # Returns `Kind` of a the instance.
```
Comment (ysbaddaden) [inline src/time/time.cr]: Typo: "of the instance"

**Comment by adlerhsieh** [comment]
Thanks. Updated. But I'm not sure if `Time::Span` will be redundant since `Span` also works for the doc parser?

**Comment by jhass** [comment]
How about a space after the `,` in the samples? `, `

**Comment by adlerhsieh** [comment]
Sure. Updated.

**Comment by jhass** [comment]
Thanks.

**Pull request merged**